src/Serialization/ScriptCollectionConverter.cs

```diff
@@ -0,0 +1,42 @@
+namespace RunScript.Serialization;
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+internal class ScriptCollectionConverter : JsonConverter<ScriptCollection>
+{
+    public override ScriptCollection Read(
+        ref Utf8JsonReader reader,
+        Type typeToConvert,
+        JsonSerializerOptions options)
+    {
+        var scripts = new ScriptCollection();
+
+        using (var jsonDoc = JsonDocument.ParseValue(ref reader))
+        using (var jsonScripts = jsonDoc.RootElement.EnumerateObject())
+        {
+            foreach (var script in jsonScripts)
+            {
+                scripts.Add(script.Name, script.Value.ToString());
```

Comment:
Maybe it's possible for a null value to end up in script.Value when deserializing.